packages/shared/src/common/__tests__/validate-npi.test.ts

```diff
@@ -0,0 +1,25 @@
+import { validateNPI } from "../validate-npi";
+import { makeNPI } from "./validate-npi";
+
+describe("validate-npi", () => {
```

Comment:
seems like we're missing a test for an invalid, but properly formatted, NPI no?

Reply:
good catch, it should be a number converted to string... the underscore there is just to make it easier to read it.

updated